src/v0/destinations/eventbridge/transform.js

```
@@ -18,7 +22,7 @@ function getResouceList(config) {
 
 function process(event) {
   // TODO: Use JSON mapping
-  if (event.destination?.Config) {
+  if (event.destination?.Config && !isEmptyObject(event.destination.Config)) {
```

Added this check so that if `event.destination?.Config` does not contain any fields; we should throw an error.

---

https://github.com/rudderlabs/rudder-transformer/pull/4435/files#diff-309f1425a405e6fa065485d37388752646daed8e6bc4517c655633f63d8d4f6fR667